src/main/java/com/parkhomovsky/bookstore/enums/Status.java

```diff
@@ -0,0 +1,7 @@
+package com.parkhomovsky.bookstore.enums;
+
+public enum Status {
```

Comment:
status of what? GIve your classes a proper names, so it is immediately understandable why the class is needed

Reply:
not resolved